### PR TITLE
Update to rc02

### PR DESCRIPTION
**Pull request by changqin26**
Hi Sebastian, 

I updated all RDF examples under the resource. Can you please check them? I am not sure about my changes in ConceptDescription and DataSpecification parts.

### CODE REVIEW

File: dataformat-rdf/src/test/resources/Complete_Example.ttl [hunk 1]
```diff
@@ -7,1387 +7,322 @@
...
+    rdfs:label "MaxRotationSpeed"^^xsd:string ;
+    <https://admin-shell.io/aas/3/0/RC02/Referable/category> "PARAMETER"^^xsd:string ;
+    <https://admin-shell.io/aas/3/0/RC02/Identifiable/id> "0173-1#02-BAA120#008"^^xsd:string ;
+	<https://admin-shell.io/aas/3/0/RC02/HasDataSpecification/dataSpecifications> [
+	   a <https://admin-shell.io/aas/3/0/RC02/Reference> ;
+	   <https://admin-shell.io/aas/3/0/RC02/Reference/type> <https://admin-shell.io/aas/3/0/RC02/ReferenceTypes/GlobalReference> ;
+	   <http://admin-shell.io/aas/3/0/RC02/Reference/value> "https://admin-shell.io/DataSpecificationTemplates/DataSpecificationIEC61360"^^xsd:string ;
+    ] .
+
+<https://admin-shell.io/DataSpecificationTemplates/DataSpecificationIEC61360> rdf:type aas:DataSpecification ;
```
[the remark this answers is not in the document]
Comment (changqin26) [reply dataformat-rdf/src/test/resources/Complete_Example.ttl:209]: we need a deeper look into "dataSpecifiction" examples. Let's do it together.

File: dataformat-rdf/src/test/resources/AAS_Reference_shortExample.ttl [hunk 2]
```diff
@@ -1,54 +1,27 @@
...
-@prefix dcterms: <http://purl.org/dc/terms/> .
-@prefix dul: <http://www.loa-cnr.it/ontologies/DUL.owl#> .
-@prefix foaf: <http://xmlns.com/foaf/spec/> .
-@prefix geo: <http://www.w3.org/2003/01/geo/wgs84_pos#> .
-@prefix om: <http://www.wurvoc.org/vocabularies/om-1.8/> .
-@prefix obda: <https://w3id.org/obda/vocabulary#> .
-@prefix owl: <http://www.w3.org/2002/07/owl#> .
-@prefix prov: <http://www.w3.org/ns/prov#> .
+@prefix : <http://www.admin-shell.io/aas/3/0/RC02/> .
+@prefix aas: <http://www.admin-shell.io/aas/3/0/RC02/> .
```
Comment (sebbader) [inline dataformat-rdf/src/test/resources/AAS_Reference_shortExample.ttl:2]: The was namespace will change with V3.0.5RC02, needs to be updated from <http://www.admin-shell.io/aas/3/0/RC02/> to <https://admin-shell.io/aas/3/0/RC02/>
This applies to all examples, not only to this one.

**Review by sebbader — CHANGES_REQUESTED** [review]
Apart of the namespace, everything looks good!

**Comment by changqin26** [comment]
Please review again. @sebbader

**Comment by sebbader** [comment]
> Please review again. @sebbader

Please check my PR here: https://github.com/changqin26/aas4j/pull/1. My SHACL validator says that the examples are good with these changes.

**Comment by changqin26** [comment]
After merging your PR, I think we are good to go. @sebbader

**Pull request merged**